sound/soc/sdca/sdca_interrupts.c

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * SDCA IRQ handler library
+ *
+ * Copyright (C) 2022-2023 Cirrus Logic, Inc. and
```

Comment:
2022-2025?

Reply:
Should really just be 2025 I think here, we havn't been working on this since 2022 :-)